html/admin/api/assets/newAssetFromType.php

```diff
@@ -27,7 +27,23 @@
 $result = $DBLIB->insert("assets", array_intersect_key( $array, array_flip( ['assets_tag','assetTypes_id','assets_notes','instances_id','asset_definableFields_1','asset_definableFields_2','asset_definableFields_3','asset_definableFields_4','asset_definableFields_5','asset_definableFields_6','asset_definableFields_7','asset_definableFields_8','asset_definableFields_9','asset_definableFields_10','assets_assetGroups'] ) ));
 
 if (!$result) finish(false, ["code" => "INSERT-FAIL", "message"=> "Could not insert asset"]);
-else finish(true, null, ["assets_id" => $result, "assets_tag" => $array['assets_tag'], "assetTypes_id" => $array['assetTypes_id']]);
+
+//Generate asset barcode
+
+$assetBarcodeData = [
+    "assetsBarcodes_value" => "A" . (is_numeric($asset['assets_tag']) ? $asset['assets_tag'] : $asset['assets_id']),
```

Comment:
Is there a reason we can't just use the tag itself?

Reply:
This follows how barcodes are generated elsewhere, happy to do this differently if you think that works best?

Reply:
That might be a hangover from how tags used to work oops sorry - I reckon we could just use the tag, that way easier for those who have existing barcodes?

Reply:
Sounds good 😆, should i update elsewhere too?

Reply:
Sounds great, sorry 😢